cdap-ui/app/cdap/components/Cloud/Profiles/Store/Provisioners.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import {createStore} from 'redux';
+import {defaultAction, composeEnhancers} from 'services/helpers';
+import {MyCloudApi} from 'api/cloud';
+
+const ACTIONS = {
+  SAVE_PROVISIONERS: 'SAVE_PROVISIONERS'
+};
+const DEFAULT_STATE = {
+  nameToLabelMap: {},
+  list: []
+};
+
+export const fetchProvisioners = () => {
+  let returnObservable$ = MyCloudApi.getProvisioners();
+  returnObservable$
+    .subscribe(
+      provisioners => {
+        store.dispatch({
+          type: ACTIONS.SAVE_PROVISIONERS,
+          payload: {
+            provisioners
+          }
+        });
+      },
+      err => {
+        console.log('Fetching provisioners failed: ', err);
```

Comment:
I don't know if it will make a difference. I am just printing it on console. A more appropriate fix would be to surface this in the UI but we don't have that design right now.